src/engine/ResultTable.h

```diff
@@ -21,7 +21,22 @@ class ResultTable {
  public:
   enum Status { FINISHED = 0, OTHER = 1 };
 
-  enum class ResultType { KB, VERBATIM, TEXT, FLOAT, STRING };
+  /**
+   * @brief Describes the type of a columns data
+   */
+  enum class ResultType {
+    // An entry in the knowledgebase
+    KB,
+    // An unsigned integer (size_t)
+    VERBATIM,
+    // An entry in the text index
```

Comment:
s/entry/offset/gc ?

Reply:
I don't see what the comment refers to. Maybe something was removed?

Reply:
The comment "An entry in the text index" but I think we are really storing a  byte offset